flexmeasures/ui/templates/base.html

```diff
@@ -649,7 +657,8 @@
 
 
     {#- Scripts used by all views (e.g. by navigation menu) -#}
-    <script src="https://ajax.aspnetcdn.com/ajax/bootstrap/3.3.7/bootstrap.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.11.8/dist/umd/popper.min.js" integrity="sha384-I7E8VVD/ismYTF4hNIPjVp/Zjvgyol6VFvRkX/vR+Vc4jQkC+hVqc2pM8ODewa9r" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/js/bootstrap.min.js" integrity="sha384-0pUGZvbkm6XF6gxjEnlmuGrJXVbNuzT9qBBavbLwCsOGabYfZo0T0to5eqruptLy" crossorigin="anonymous"></script>
```

Comment:
I don't know why we load bootstrap twice, actually. Do you?

We'll have to check (later) how to load it only once ...

Reply:
not sure as well but Popper.js is a library used for positioning tooltips and popovers while The Bootstrap JavaScript file provides the interactivity and dynamic features for Bootstrap components.

Reply:
replaced the two links , with the boostrap bundle